### PR TITLE
105 graph anttns

### PR DESCRIPTION
**Pull request by sputnik792**
added graph columns to display the annotation messages,
slight change in the indexing.

### CODE REVIEW

**Comment by TrevorEdwards** [comment]
Code looks good, but I think the original intention of the issue was also to have a datepicker https://eonasdan.github.io/bootstrap-datetimepicker/ for adding new annotations, since it just assumes the present as is. That might require slightly changing a back-end api, but I don't think it should be too much.

**Comment by TrevorEdwards** [comment]
Datepicker would be optional, we could assume present by default. That's just to make it so if we do a FB ad campaign and forget to make an annotation, we can add it back in later.

File: views/admin/stats.jade [hunk 1]
```diff
@@ -44,4 +47,5 @@ block append footer
     script(type='text/javascript').
         var users =!{JSON.stringify(users)}
         var annotations =!{JSON.stringify(annotations)}
-    script(src="/js/stats.js")
+    script(src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.6.4/js/bootstrap-datepicker.min.js")
```
Comment (mahsu) [inline views/admin/stats.jade]: There should be a `footer` block you can append these scripts to.

**Comment by TrevorEdwards** [comment]
merge develop into this branch before later merge

File: views/admin/stats.jade (hunk 1 above)
Comment (TrevorEdwards) [inline views/admin/stats.jade]: It's in that, i think github interface made it confusing

File: public/js/stats.js [hunk 2]
```diff
@@ -62,32 +52,37 @@ var chart = c3.generate({
...
+    $("#send-ann").on('click', function () {
+        var that = this;
+        $.ajax({
+            method: "POST",
+            url: "/api/admin/annotate",
+            data: {
+                time: $("#datepicker").datepicker('getDate'),
+                annotation: $("#annotation").val()
+            },
+            error: function () {
```
Comment (TrevorEdwards) [inline public/js/stats.js]: Put in default alert handler

File: routes/api/admin.js [hunk 3]
```diff
@@ -892,6 +892,9 @@ function deleteAnnouncement(req, res, next) {
  *
  */
 function annotate(req, res, next) {
+    // format the date
+    console.log(req.body.time);
```
Comment (TrevorEdwards) [inline routes/api/admin.js]: remove console.log and new line

File: views/admin/stats.jade (hunk 1 above)
Comment (mahsu) [inline views/admin/stats.jade]: ah i see it now my bad

**Pull request merged**